API Blueprint Specification.md

```diff
@@ -277,8 +280,12 @@ What sections can be nested and where depends upon the section in case, as descr
 # II. Sections Reference
 > **NOTE:** Sections marked as "Abstract" serve as a base for other sections and as such they **cannot** be used directly.
 
+<!-- BEGIN Abstract -->
+
+# Abstract
+
 <a name="def-named-section"></a>
-## 1. Named section
+## Named section
```

Comment:
You have removed numbers from headings in "II. Sections Reference" but left them for headings in "I. API Blueprint Language" I would say, for consistency, the numbers should stay in the headings. Thoughts?